src/services/services.ts

```diff
@@ -6634,14 +6640,20 @@ namespace ts {
                     getReferencesForStringLiteralInFile(sourceFile, type, possiblePositions, references);
                 }
 
+                const symbol = typeChecker.getSymbolAtLocation(node);
+
+                const displayParts = symbol ? getSymbolDisplayPartsDocumentationAndSymbolKind(
+                    symbol, node.getSourceFile(), getContainerNode(node), node).displayParts : undefined;

```

Comment:
Yeah, I think you're right. I'll make the change. This also removes the case where display parts can be undefined, so I'll make that change too.